recipe/meta.yaml

```diff
@@ -7,12 +7,12 @@ package:
   version: {{ version|replace("-", "_") }}
 
 source:
-  url: https://github.com/EDIorg/EMLassemblyline/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 1596d2b2dc00049103746d9cd563cbc8f36e5b76991c9305b8b2d2905fbfb924
+  url: https://github.com/yvanlebras/EMLassemblyline/archive/refs/heads/{{ version }}-dev.zip
+  sha256: 38e5a8fd75f76ad118f4eb89da7fbf604edbeae47dfccb2ba5958ed893756beb
 
 build:
   merge_build_host: true  # [win]
-  number: 1
+  number: 2
```

Comment:
This should be a version change, not a build bump. Change the version to `'3.5.5-dev'` in the Jinja variable and this should go to 0.

Reply:
I updated to 0 after changing the url to a tag version of our dev repository